src/wizards/services/WizardService.ts

```diff
@@ -4,6 +4,8 @@ import { Router } from "aurelia-router";
 import { STAGE_ROUTE_PARAMETER } from "wizards/tokenSwapDealWizard/dealWizardTypes";
 import { Rule, validateTrigger, ValidationController, ValidationControllerFactory } from "aurelia-validation";
 import { PrimeRenderer } from "resources/elements/primeDesignSystem/validation/primeRenderer";
+import { WizardManager } from "wizards/tokenSwapDealWizard/wizardManager";
```

Comment:
This must be removed.  The wizardService is meant to be deal-independent.  Further, it would create a circular dependency.

Reply:
[refac(wizard): better name for key for wizardStates map](https://github.com/PrimeDAO/prime-deals-dapp/pull/227/commits/e4fcf10ae38cc8df6d5bfec5ce0f89073f3b33e8)